pkg/plugins/plugins.go

```diff
@@ -228,9 +230,11 @@ func (pm *PluginManager) scan(pluginDir string, requireSigned bool) error {
 		}
 
 		pm.log.Debug("Found plugin", "id", plugin.Id, "signature", plugin.Signature, "hasRoot", plugin.Root != nil)
-		if !scanner.validateSignature(plugin) {
-			pm.log.Debug("Not adding plugin since it lacks a valid signature", "id", plugin.Id,
-				"signature", plugin.Signature)
+		signingError := scanner.validateSignature(plugin)
+		if signingError != nil {
+			pm.log.Warn("Plugin has signature errors", "id", plugin.Id,
```

Comment:
Guess the errors are already logged with error level somewhere so I guess that was the reason it used debug level here before. Maybe @aknuds1 has input on this. 

Would maybe change message to "failed to validate plugin signature"

Reply:
Yeah actually was just an oversight - will return to Debug and update the message 👍